x-pack/elastic-agent/docs/tab-widgets/download.asciidoc

```diff
@@ -0,0 +1,95 @@
+// tag::deb[]
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {agent} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+IMPORTANT: To simplify upgrading to future versions of {agent}, we recommended
+that you use the tarball distribution instead of the DEB distribution.
+
+["source","sh",subs="attributes"]
+----
+curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-{version}-amd64.deb
+sudo dpkg -i elastic-agent-{version}-amd64.deb
+----
+
+endif::[]
+// end::deb[]
+
+// tag::rpm[]
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {agent} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+IMPORTANT: To simplify upgrading to future versions of {agent}, we recommended
+that you use the tarball distribution instead of the RPM distribution.
+
+["source","sh",subs="attributes"]
+----
+curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-{version}-x86_64.rpm
+sudo rpm -vi elastic-agent-{version}-x86_64.rpm
+----
+endif::[]
+// end::rpm[]
```

Comment:
added. I made some other changes to the rpm/deb part. I think it will be easier for you to confirm the changes after they are merged and building in the docs.